.github/workflows/ci-e2e-checks.yaml

```diff
@@ -9,25 +9,25 @@ on:
       - master
 
 jobs:
- 
+
   e2e-tests:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         k8s-version:
+          - v1.22
+          - v1.21
           - v1.20
           - v1.19
-          - v1.18
-          - v1.17
         include:
+          - k8s-version: v1.22
+            kind-node-image: kindest/node:v1.22.0@sha256:b8bda84bb3a190e6e028b1760d277454a72267a5454b57db34437c34a588d047
```

Comment:
**question: (non-blocking)** Is there a reason to use v1.22.0 for this node image?  It looks like the PR bumps KinD to v0.12 which suggests `kindest/node:v1.22.7@sha256:1dfd72d193bf7da64765fd2f2898f78663b9ba366c2aa74be1fd7498a1873166` is a good node image.

additional relevant images if we want to bump them here. 
- `kindest/node:v1.21.10@sha256:84709f09756ba4f863769bdcabe5edafc2ada72d3c8c44d6515fc581b66b029c`
- `kindest/node:v1.20.15@sha256:393bb9096c6c4d723bb17bceb0896407d7db581532d11ea2839c80b28e5d8deb`
- `kindest/node:v1.19.16@sha256:81f552397c1e6c1f293f967ecb1344d8857613fb978f963c30e907c32f598467`

Reply:
Good catch, will update to recommended images based on KinD version.

Reply:
Bumped.